File: httpsnippet/src/test/resources/output/node/request/full.js
var request = require("request");

var jar = request.jar();
jar.setCookie(request.cookie("foo=bar"), "null");
jar.setCookie(request.cookie("bar=baz"), "null");

var options = {"headers":{"content-type":"application/x-www-form-urlencoded","accept":"application/json"},"qs":{"foo":["bar","baz"],"baz":["abc"]},"method":"POST","jar":jar,"url":"http://mockbin.com/har?key=value","forms":{"foo":"bar"}};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: httpsnippet/src/test/resources/output/ruby/nethttp/full.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?key=value")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["Cookie"] = "foo=bar;bar=baz"
request["content-type"] = "application/x-www-form-urlencoded"
request["accept"] = "application/json"

response = http.request(request)
puts response.read_body